src/components/dropdown/dropdown.tsx

```diff
@@ -194,7 +194,7 @@ export class Dropdown implements InteractiveComponent {
             ref={this.setScrollerEl}
           >
             <div hidden={!this.active}>
-              <slot />
+              <slot onSlotchange={this.updateItems} />
```

Comment:
Some ideas:

Instead of calling `getSlotted` in `render` where most components do, we could just call it `onSlotChange`. Then we wouldn't have to query for the slot since we would have its reference from the event. It would just be getting its assignedElements and storing them on the instance for usage.